src/modules.js

```diff
@@ -213,39 +235,62 @@ if (__analytics.amplitude?.isEnabled) {
         analyticsApi.defaults.headers.Authorization = `Bearer ${token}`;
         await analyticsApi.post(publishURL, payload);
       };
-      initializeApp(firebaseConfig);
```

Comment:
@AlanLes before your changes we use `initializeApp(firebaseConfig)` and pass 
```
firebaseConfig: {
  apiKey: process.env.ANALYTICS_FIREBASE_API_KEY,
  authDomain: process.env.ANALYTICS_FIREBASE_AUTH_DOMAIN,
  appId: process.env.ANALYTICS_FIREBASE_APP_ID,
},
```
I catch `const firebaseApp = initializeApp({})` in line 258 but... where we pass firebaseConfig to sign in anonymously